### ISSUE TITLE
DietPi Buster + Raspberry Pi 4 works well!

### ISSUE THREAD

**Issue by raspberry-pi-maker** [hzeller/rpi-rgb-led-matrix#991]
When I used Rasbian Buster with Rpi4 issue #483 occcurs.  I have to use an old Rasbian Jessie. But the Rpi 4 doesn't support Jessie. As a result, I have to use Rpi 3 instead of 4.
Yesterday, I tested DietPi Buster with Rpi 4. The RGB LED matrices display is good. No flickering.
I've write the test process in my blog (https://iot-for-maker.blogspot.com/2020/02/led-5-raspberry-pi-4-dietpi-buster.html).  I hope this can help you who want to use Rpi 4.

**Comment by mikecann** [comment]
This is good to know. Im about to embark on a project involving 6 x 64x64 panels using a pi 4 and the Electrodragon HAT. 

I was hoping to power them from battery, how likely do you think that is?

**Comment by raspberry-pi-maker** [comment]
When all the RGB LED pixels are on, max power might be needed.
However, this rarely happens. 
Of course, it's a good idea to have a power supply that can handle the maximum power.
In my case, considering the cost, there was no big problem using 5V10A to handle P3 64X32 4EA.

**Comment by marcmerlin** [comment]
Thanks @raspberry-pi-maker I was about to ask @hzeller if there was any benefit to using an rPi4 over a 3?
From what I understand the rPi3 is already able to output faster than the panels are able to absorb the input and that while 256x256 is possible with an electrodragon active-3 board, it's pushing the limits of what can be done with 3 channels/1 Pi (refresh rate and color depth are not great).
My assumption so far is that an rPi4 doesn't change this and 256x256 is still more or less the limit.
I opened 
https://github.com/hzeller/rpi-rgb-led-matrix/pull/971  https://github.com/hzeller/rpi-rgb-led-matrix/pull/971/commits/57031074130bfe49dc2ed9d0191febaf0ab364d1 to write this to the best of my understanding.

**Comment by mikecann** [comment]
@marcmerlin I actually found that the pi4 was cheaper than the pi3 thats why I chose it. Plus the extra power lets me run more complex code or run in a higher level language or something.

**Comment by hzeller** [comment]
Yes, the faster internal processing makes a Pi4 more desirable than a Pi3 even if the clock to the matrix is at their limit.

**Comment by raspberry-pi-maker** [comment]
I agree with @hzeller . 
Even though the rPi3 has enough processing power to GPIO handling, making the contents to display is also important.
I like to use OpenCV to create the contents. The greater the processing power, the faster the OpenCV's processing speed. Therefore,  rPi4 is more advantageous than rPi3 when working with OpenCV such as video display.

**Comment by marcmerlin** [comment]
With apologies to everyone, this project is doing a one time cleanup of 
all the bugs that have not been addressed and closed in the last 10 years.

If you filed an issue to ask for help, please do not re-open this one or a new
bug, but go here instead: https://rpi-rgb-led-matrix.discourse.group/

Please be mindful that this project does not have any full time maintainers
or anyone paid. You should not expect personal help for your own projects or
answers to programming questions but you can ask for help with panels that
aren't working, setup questions, or anything that may be a bug here:
https://rpi-rgb-led-matrix.discourse.group/
If you need programming help and are willing to pay someone for it, please
state so in your message, and someone may be interested.

If you did report an actual bug that is still not fixed today, please accept
our apologies for this being mass closed, you can absolutely send a followup
and re-open the issue, or open a new up to date one as applicable.
Note that we very much welcome help, so if you can help fix your bug, you will
be a superstar :)

If your bug was linked to a pull request that was not resolved, please update
the pull request and the issue, and we will do our best to review them. We do
thank you for your undertanding that we have very limited free time, so any
extra work you do along with your bug report or PR request will be very much
appreciated and improve the odds that they will be looked at.

If you want to help contribute as a python or C# or any other language bindings,
you are also a superstar, please state so in the issue and we'll do our best to
give you the access and reviews you need.

For anything related to PWM panel support, please make sure you read README and
then refer to https://github.com/hzeller/rpi-rgb-led-matrix/issues/466 . 
Please do not comment if you're haven't read everything and/or if you're just
asking when your panel is going to be supported. Those bugs are if you're 
able to contribute to the effort.

For rPI5: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1603

For anything python, please link your bug to: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1828
For anything C#, please link your bug to: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1828
